mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -89,6 +89,9 @@ class GaussianMixtureModel private[ml] (
   extends Model[GaussianMixtureModel] with GaussianMixtureParams with MLWritable
   with HasTrainingSummary[GaussianMixtureSummary] {
 
+  @Since("3.0.0")
+  val numFeatures: Int = gaussians.head.mean.size
```

Comment:
This PR seems to add at least 4 `numFeatures` instance. Could you add this into the PR description explicitly?